examples/complete/variables.tf

```diff
@@ -31,3 +31,15 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "scope" {
```

Comment:
remove this variable from the example - you already have a local set in the code, so we just use that for the example